.github/workflows/ci.yml

```diff
@@ -64,6 +64,15 @@ jobs:
             echo "CARGO_INCREMENTAL=${{ env.CARGO_INCREMENTAL }}"
             echo "RUST_BACKTRACE=${{ env.RUST_BACKTRACE }}"
 
+      - name: Change target output directory on Windows
+        # Windows doesn't have enough space on the D: drive, so we redirect the build output to the
+        # larger C: drive.
+        # TODO: Remove this workaround once the build is more efficient.
+        if: matrix.os == 'windows-latest'
+        run: |
+          mkdir C:\zebra-target
+          echo "CARGO_TARGET_DIR=C:\zebra-target" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+
```

Comment:
I think this is fine - I was going to suggest changing the working directory for the whole job to be the C: location, but that would have to go earlier and may be unnecessary because while the commands may run on D:, the target is still C:, which is where we need the space.

Hopefully yes we can be freed from this restriction with a leaner build (feels like 12GB should be enough! 🥴)

Reply:
That was an interesting issue.